### PR TITLE
chore: contributor-friendly error message

### PR DESCRIPTION
**Pull request by SkyFan2002**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Closes #issue

### CODE REVIEW

File: src/meta/client/src/lib.rs [hunk 1]
```diff
@@ -37,7 +37,10 @@ pub static METACLI_COMMIT_SEMVER: Lazy<Version> = Lazy::new(|| {
 
     let semver = semver.strip_prefix('v').unwrap_or(semver);
 
-    Version::parse(semver).unwrap_or_else(|e| panic!("Invalid semver: {:?}: {}", semver, e))
+    Version::parse(semver).expect(
+        "run `git fetch --tags https://github.com/datafuselabs/databend` to slove this error,
```
Comment (BohuTANG) [inline src/meta/client/src/lib.rs:41]: Not need the repo name https://github.com/datafuselabs/databend?

Comment (SkyFan2002) [reply src/meta/client/src/lib.rs:41]: IMHO, new contributors may create their own fork and have not set this repository as upstream yet, so fetch with repo name is a general method to fetch newest tags.

Comment (BohuTANG) [reply src/meta/client/src/lib.rs:41]: When the user meet the error, it means that he has clone the repo but not fetch the tags, now only run `git fetch --tags` is ok.

**Review by drmingdrmer — APPROVED** [review]

**Comment by BohuTANG** [comment]
Is this PR ready for review?

**Review by BohuTANG — APPROVED** [review]

**Pull request merged**